tc/core/polyhedral/schedule_tree.h

```diff
@@ -344,6 +344,20 @@ struct ScheduleTree {
         detail::ScheduleTreeType::Sequence, std::forward<Args>(args)...);
   }
 
+  // Is "tree" a mapping filter that maps identifiers of the type provided as
+  // template argument?
+  template <typename MappingType>
+  static bool isMappingTo(const detail::ScheduleTree* tree) {
```

Comment:
Does it have to be a _member_ function?  I would just put it into `schedule_transforms.h` together with `activeDomainPoints` and schedule collectors.